guacamole/src/main/frontend/src/app/index/indexModule.js

```diff
@@ -17,22 +17,37 @@
  * under the License.
  */
 
+require('angular-module-shim.js');
+require('relocateParameters.js');
+
+require('angular-translate-interpolation-messageformat');
+require('angular-translate-loader-static-files');
+
 /**
  * The module for the root of the application.
  */
 angular.module('index', [
+
+    require('angular-route'),
```

Comment:
Why are these here?

Reply:
These are the external AngularJS libraries that we depend on. It's a convention with AngularJS libraries within NPM to simply export the name of the relevant AngularJS module, so `require('angular-route')` here returns a JavaScript string that is the name of the AngularJS module providing routing.